clang/test/Preprocessor/sycl-macro2.cpp

```diff
@@ -0,0 +1,4 @@
+// RUN: %clang -c %s -fsycl -E -dM -o - | FileCheck --check-prefix=CHECK-MSVC %s
+// REQUIRES: system-windows
```

Comment:
See below.  Use triples if you care about platform.

Reply:
@erichkeane, thanks I will just remove this test.   Alexey feels it is redundant.